src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -106,14 +104,15 @@ export class DefaultSamLocalInvokeCommand implements SamLocalInvokeCommand {
                     }
                 },
                 onError: (error: Error): void => {
-                    this.channelLogger.error('AWS.samcli.error', 'Error running command "{0}": {1}', samCommandName, error)
+                    this.channelLogger.error(
+                        'AWS.samcli.error',
+                        'Error running command "{0}": {1}',
+                        samCommandName,
+                        error
+                    )
                     reject(error)
                 },
             })
-
-            if (!params.isDebug || this.debuggerAttachCues.length === 0) {
-                resolve()
```

Comment:
not sure what the purpose of this was. If the caller of `invoke()` does not want to wait they can just omit `await`.